stubs/reportlab/reportlab/pdfbase/ttfonts.pyi

```diff
@@ -142,9 +143,42 @@ class TTFont:
     encoding: Incomplete
     state: Incomplete
     def __init__(
-        self, name, filename, validate: int = 0, subfontIndex: int = 0, asciiReadable: Incomplete | None = None
+        self,
+        name,
+        filename,
+        validate: int = 0,
+        subfontIndex: int = 0,
+        asciiReadable: Incomplete | None = None,
+        shaped: Incomplete | None = None,
     ) -> None: ...
     def stringWidth(self, text, size, encoding: str = "utf8"): ...
     def splitString(self, text, doc, encoding: str = "utf-8"): ...
     def getSubsetInternalName(self, subset, doc): ...
     def addObjects(self, doc) -> None: ...
+    @property
+    def hbFace(self) -> Incomplete | None: ...
+    def hbFont(self, fontSize: float = 10): ...
+    @property
+    def isShaped(self) -> bool: ...
+    @isShaped.setter
+    def isShaped(self, v) -> None: ...
+    def pdfScale(self, v): ...
+    def unregister(self) -> None: ...
+
+class ShapedFragWord(list[Any]): ...
```

Comment:
We either need an explanation (comment) for `Any` here or an `Incomplete`.

Reply:
From what I can see in the source, this class is basically only used for a specific purpose, although I haven't looked too deep into it. I think using either `Incomplete` (and later finding out what that purpose is) or making it generic is fine.